clients/ui/README.md

```diff
@@ -58,56 +58,75 @@ The following environment variables are used to configure the deployment and dev
 * **Possible Values**: `docker`, `podman`, etc.
 * **Example**: `CONTAINER_TOOL=docker`
 
-### `IMG_BFF`
+### `IMG_UI`
 
-* **Description**: Specifies the image name and tag for the Backend For Frontend (BFF) service.
-* **Default Value**: `model-registry-bff:latest`
-* **Example**: `IMG_BFF=model-registry-bff:latest`
+* **Description**: Specifies the image name and tag for the UI (with BFF).
+* **Default Value**: `model-registry-ui:latest`
+* **Example**: `IMG_UI=model-registry-bff:latest`
 
-### `IMG_FRONTEND`
+### `IMG_UI_STANDALONE`
 
-* **Description**: Specifies the image name and tag for the frontend service.
-* **Default Value**: `model-registry-frontend:latest`
-* **Example**: `IMG_FRONTEND=model-registry-frontend:latest`
+* **Description**: Specifies the image name and tag for the UI (with BFF) in **standalone mode**, used for local kind deployment.
+* **Default Value**: `model-registry-ui-standalone:latest`
+* **Example**: `IMG_UI_STANDALONE=model-registry-bff:latest`
```

Comment:
```suggestion
* **Example**: `IMG_UI_STANDALONE=quay.io/personal/model-registry-ui:latest` 
```
For these examples I would do something like this to show how you can do it

Reply:
Ok, I've formatted part of the docs, could you re-review it?